evalml/tests/component_tests/test_estimators.py

```diff
@@ -359,3 +359,30 @@ def test_estimator_fit_predict_and_predict_proba_respect_custom_indices(
         )
     X_pred = estimator.predict(X)
     pd.testing.assert_index_equal(X_original_index, X_pred.index, check_names=True)
+
+
+@pytest.mark.parametrize("estimator_class", _all_estimators())
```

Comment:
Should we delete any of the previous tests? I think we could remove all the tests don't check equality (i.e the estimators that don't support feature importance) since there's some redundancy between these tests.

Reply:
Indeed, I can see this going both ways.  I can see tests here in `test_estimators` existing for the purpose of determining that no `feature_importance()` sends back NaN.  If we wanted to do any assertion on the shapes of feature importances relative to the input datasets, now and here would be the place to do it.  I think if we wanted, for some reason, to test the specific `feature_importance()` of an individual estimator and make sure that it calculates the right values, we'd want to do it in that specific module.